source/game.programmeproducer.bmx

```diff
@@ -314,7 +317,7 @@ Type TProgrammeProducer Extends TProgrammeProducerBase
 				'if person print "would reuse: " + person.GetFullName()
 				'slight chance to ignore the given one?
 				'and to look for a new one
-				If person And RandRange(0,100) < 10 Then person = Null
+				If person And (RandRange(0,100) < 10 Or person.GetAge() < 20) Then person = Null
```

Comment:
Waere es OK fuer dich, wenn wir diese Ueberpruefung in ein "UsableForNonPlayerProductions()" oder so auslagern?
(oder wie man das dann am Besten nennt)

Mich stoert ein wenig die Verteilung dieser "GetAge() < 20" -Zeilen.

Reply:
Ja - quick and dirty. Schau' ich mir nochmal an.

Reply:
rausgezogen